addons/account/static/src/components/many2x_tax_tags/many2x_tax_tags.js

```diff
@@ -16,16 +16,22 @@ export class Many2XTaxTagsAutocomplete extends Many2XAutocomplete {
     async loadOptionsSource(request) {
         // Always include Search More
         let options = await super.loadOptionsSource(...arguments);
-        if (!options?.slice(-1)[0].classList?.includes("o_m2o_dropdown_option_search_more")) {
+        if (!options.slice(-1)[0]?.cssClass?.includes("o_m2o_dropdown_option_search_more")) {
             options.push({
                 label: this.SearchMoreButtonLabel,
-                action: this.onSearchMore.bind(this, request),
-                classList: "o_m2o_dropdown_option o_m2o_dropdown_option_search_more",
+                onSelect: this.onSearchMore.bind(this, request),
+                cssClass: "o_m2o_dropdown_option o_m2o_dropdown_option_search_more",
             });
         }
         return options;
     }
 
+    mapRecordToOption(record) {
+        let option = super.mapRecordToOption(...arguments);
```

Comment:
`option` is never re-assigned so it could be a `const`

Reply:
Yeah I know, but I since I change a key value, I prefer to use `let`. For me it's just more intuitive that something is changing inside (btw I think it's crazy to be able to change on object value when it's `const`)
but this is just personal preference. If there's a downside to `let` or you feel strongly about it, I'll change it.

Reply:
I won't block the PR as it's also a personal preference (even if it's against this ES lint [rule](https://eslint.org/docs/latest/rules/prefer-const)) but good to me :+1:

Reply:
Why even having a `const` (or a `let`) :smirk: 
```js
mapRecordToOption(record) {
    return {
        ...super.mapRecordToOption(...arguments),
        label: record.name,
    }
}
```